### PR TITLE
Script frame sequence number matching

### PR DESCRIPTION
**Pull request by Erol444**
2nd stage NN should now get the same frame as the 1st stage NN by using frames sequence number in the scripting node.

Working confirmed [on Discord](https://discord.com/channels/790680891252932659/795743863519576105/871795031781040148).

### CODE REVIEW

File: gen2-age-gender/main.py [hunk 1]
```diff
@@ -74,36 +74,66 @@ def create_pipeline():
...
+l = [] # List of images
+# So the correct frame will be the first in the list
+# For this experiment this function is redundant, since everything
+# runs in blocking mode, so no frames will get lost
+def remove_prev_frame(seq):
+    for rm, frame in enumerate(l):
+        if frame.getSequenceNum() == seq:
+            # node.warn(f"List len {len(l)} Frame with same seq num: {rm},seq {seq}")
+            break
+    for i in range(rm):
```
Comment (VanDavv) [inline gen2-age-gender/main.py:93]: on the first iteration, if `l` is empty, the `rm` variable would not be initialized and throw error

**Review by VanDavv — CHANGES_REQUESTED** [review]
Nice,

File: gen2-age-gender/main.py (hunk 1 above)
Comment (Erol444) [reply gen2-age-gender/main.py:93]: Thanks and thanks for noticing this! I'm not sure how inferencing could have arrived to the script node before the frame, but better to ba safe, especially when other users might edit this script.

**Review by VanDavv — APPROVED** [review]
Just gave it a try using 2.9.0.0 API and works great, thanks!

**Pull request merged**